### DISCUSSION TITLE
[部署求助]修改了Dockerfile和Dockerfile.database之后如何部署

### DISCUSSION THREAD

**Discussion by King-WKR** [lobehub/lobehub#4956] [Self Hosting]
我想做个带有clerk的lobechat，修改了[Dockerfile]和[Dockerfile.database]。
向各位大佬请教一下如何部署，改后的项目
目前我已经clone到我的linux上了，但是docker一直起不来
![image](https://github.com/user-attachments/assets/c390f77b-1be1-4000-a241-8d383368fea7)

**Comment by SpeedupMaster** [comment]
要自己构建镜像

**Reply by King-WKR** [reply]
啊，我以为只需要修改就好

**Reply by SpeedupMaster** [reply]
> 啊，我以为只需要修改就好

我之前是在windows上构建然后导入到VPS上的，VPS性能不行，构建有点慢

**Comment by SpeedupMaster** [comment]
#4929 不然你看下用Dokploy部署

**Reply by King-WKR** [reply]
我研究下这个方式吧，似乎便捷点，感谢大佬
